2-exploring-and-comparing-different-llms/README.md

```diff
@@ -1,23 +1,24 @@
 # Exploring and comparing different LLMs
 
-## Introduction
+With the previous lesson, we have seen how Generative AI is changing the technology landscape, how Large Language Models (LLMs) work and how a business - like the Edu4All startup - can apply them to their use cases and grow! In this chapter, we're looking to compare and contrast different types of large language models, LLMs to understand their pros and cons.
 
-With the previous lesson, we have seen how Generative AI is changing the technology landscape, how Large Language Models (LLMs) work and how a business - like the Edu4All startup - can apply them to their use cases and grow!
+## Introduction
 
 The next step in our startup's journey is exploring the current landscape of Large Language Models (LLMs) and understanding which are suitable for our use case.
 
 This lesson will cover:
 
-- Different types of LLMs in the current landscape
-- Testing, iterating, and comparing different models for your use case in Azure
-- How to deploy an LLM
+- Different types of LLMs in the current landscape.
+- Testing, iterating, and comparing different models for your use case in Azure.
+- How to deploy an LLM.
 
 ## Learning Goals
 
 After completing this lesson, you will be able to:
-- Select the right model for your use case
-- Understand how to test, iterate, and improve performance of your model
-- Know how Business deploy models
+
+- Select the right model for your use case.
+- Understand how to test, iterate, and improve performance of your model.
+- Know how Business deploy models.
 
 ## Understand different types of LLMs 
 
```

Comment:
```suggestion

There are many different types of LLM models, your choice of model depends on what you aim to use them for, your data, how much you're ready to pay and more. 

Depending on if you aim to use the models for text, audio, video, image generation and so on, you might opt for a different type of model.

- **Audio and speech recognition**. For this purpose, Whisper-type models are a great choice as they're general-purpose and aimed at speech recognition. It's trained on diverse audio and can perform multilingual speech recognition. As an example, you can use everything from a cheaper, but capable model like curie to the more costly but performat davinci type model. Learn more about [Whisper type models here](https://platform.openai.com/docs/models/whisper). 
- **Image generation**. For image generation, DALL-E and Midjourney are two very known choices. DALL-E is offered by Azure OpenAI. [Read more about DALL-E here](https://platform.openai.com/docs/models/dall-e) and also in Chapter 9 of this curriculum 
- **Text generation**. Most models are trained on text generation and you have a large variety of choices from GPT-3, GPT-3.5 to GPT-4. They come at different costs with GPT-4 being the most expensive. It's worth looking into the [Azure Open AI playground](https://oai.azure.com/portal/playground) to evaluate which models best fit your needs in terms of capability and cost.

Selecting a model means you get some basic capabilties, that might not be enough however. Often you have company specific data that you somehow need to tell the LLM about. There are a few different choices on how to approach that, more on that in the upcoming section.
```

@carlotta94c integrated this has a high level intro text to the above h2 header, and removing the text from the area it came from, thoughts on that change?

Reply:
@softchris yes I think this is great as an introduction :) The only comment I have is I would remove GPT-3 from the list, since it's deprecated. Otherwise, looks good! Thanks